stress-ng/cmd.sh

```diff
@@ -45,28 +46,34 @@ echo "allowed cpu list: ${cpulist}"
 uname=`uname -nr`
 echo "$uname"
 
-cpulist=`convert_number_range ${cpulist} | tr , '\n' | sort -n | uniq`
+if [ "${CMDLINE}" == "" ]; then
```

Comment:
Please change to the following to be consistent with the other tests in this script:
`if [[ -z "${CMDLINE}" ]]; then`